maps/site53/site53-2.dmm

```diff
@@ -30713,6 +30710,9 @@
 	},
 /turf/simulated/floor/tiled/monotile,
 /area/site53/lowertrams/hub)
+"qVm" = (
```

Comment:
This all looks good. I uh, I really have no idea why this map edit is here, it seems to change nothing. Please remove it.

Reply:
I've tried it won't go away

Reply:
Take the file from the commit before and just paste it into your repo. You can view the history of the repository at specific points in history, like a wiki

Reply:
My main concern is this fucking up and causing conflicts for people with actual map PRs.

Reply:
It only effects one tile

Reply:
It doesn't matter, it's an unnecessary change. It's my job to enforce standards, as pedantic as they may seem at times. Hell I'll do it myself if you want, but you'll have to wait

Reply:
I don't have the ability to fix it right now, and my next pr is messing with that area

Reply:
> I don't have the ability to fix it right now, and my next pr is messing with that area

download the codebase as a zip and get the area from there using SDMM